src/containers/account/reducers/regenerateCredentialReducer.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * @flow
+ */
+
+import { Map } from 'immutable';
+import { RequestStates } from 'redux-reqseq';
+import type { SequenceAction } from 'redux-reqseq';
+import { PrincipalsApiActions } from 'lattice-sagas';
+
+import { REQUEST_STATE } from '../../../core/redux/constants';
+
+const { REGENERATE_CREDENTIAL, regenerateCredential } = PrincipalsApiActions;
+
+export default function reducer(state :Map, action :SequenceAction) {
+
+  return regenerateCredential.reducer(state, action, {
+    REQUEST: () => state
+      .setIn([REGENERATE_CREDENTIAL, REQUEST_STATE], RequestStates.PENDING)
+      .setIn([REGENERATE_CREDENTIAL, action.id], action),
+    SUCCESS: () => {
+      const storedAction = state.getIn([REGENERATE_CREDENTIAL, action.id]);
+      if (storedAction) {
```

Comment:
No change required, just a small note. Since you're not using `storedAction`, you can just do `state.hasIn` like you do in the `FAILURE` block.